packages/Notification/Notification.jsx

```diff
@@ -162,6 +192,20 @@ Notification.propTypes = {
    * @since 1.2.0
    */
   dismissibleA11yLabel: PropTypes.string,
+  /**
+   * A callback function to be run on click of the dismissible icon.
+   * Requires `dismissible={true}`
+   *
+   * @since 1.3.0
+   */
+  onExit: PropTypes.func,
+  /**
+   * A callback function to be run on when the Notification has been fully dismissed and unmounted
```

Comment:
Typo: A callback function to be run ~on~ when the Notification has been fully dismissed and unmounted